.github/workflows/delivery-ubuntu.yml

```diff
@@ -68,44 +68,7 @@ jobs:
             ###
 
           - name: Deliver bionic
-            if: matrix.target == 'bionic'
-            uses: docker://ubuntu:bionic
-            with:
-              entrypoint: .github/workflows/delivery/ubuntu/deliver.sh
-            env:
-              DEBIAN_FRONTEND: "noninteractive"
```

Comment:
@AidanDelaney, don't we need the env variables?

Reply:
The env vars are still there.  I think it's just how GH presents the diff.